recipes/kubernetes-client-python/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "2.0.0" %}
+
+package:
+  name: kubernetes-client-python
```

Comment:
Read your [discussion]( https://github.com/conda-forge/staged-recipes/pull/3125#discussion-diff-121696928R4 ) in the outdated diff.

I agree that the PyPI name is suboptimal. That said, if we call it something vastly different from the PyPI name, this will cause confusion amongst users. Encountered this recently with `fftw` (where I thought such confusion shouldn't occur as the Python package is `pyfftw`). Also this is a semi-regular issue with `graphviz` where there is the Python library (we call `python-graphviz`), and the C/C++ library and CLI (we call `graphviz`) sharing the same name.

That said, if there is a reasonable chance of us having a `kubernetes` conda package, which could be possible, would go to the more standard `python-` prefixing. This is used pretty much throughout conda-forge to handle these issues. If we deviate from that strategy, it will likely cause more confusion.

Happy to discuss further.